{"uuid":"3941dfd1-2646-4fe9-946c-b451f087b446","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["6dda3299-f9ec-4246-b936-e4c75561ba5b"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696886284601,"stop":1696886284602},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696886285086,"stop":1696886285086},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696886285204,"stop":1696886285205},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"6b7d402f-1f9b-4642-8fe6-7184ec983972-attachment","type":"image/png"}],"parameters":[],"start":1696886287960,"stop":1696886288475}],"start":1696886284181,"stop":1696886288475}